src/main/java/st/ratpack/auth/CachingTokenValidator.java

```diff
@@ -21,39 +21,23 @@ public CachingTokenValidator(TokenValidator upstreamValidator) {
 		cache = Caffeine.<String, Promise<Optional<OAuthToken>>>newBuilder()
 			.maximumSize(10000L)
 			.expireAfterWrite(5L, TimeUnit.MINUTES)
+			.recordStats()
 			//			.executor(Execution.current().getEventLoop())  Don't do this it makes Ratpack hang.
 			.build();
```

Comment:
In general prefer using `LoadingCache`.

```java
cache = Caffeine.newBuilder()
    .expireAfterWrite(5, TimeUnit.MINUTES)
    .maximumSize(10_000)
    .recordStats()
    .build(this::loadCache);
```

Then the lookup becomes,
```java
return cache.get(token);
```

Why is gathering statistics enabled but not exposed? There is a small penalty, but its generally not a big deal.